.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -63,3 +65,13 @@ checked:
 - [ ] Workflow Diagrams: Ensure workflow diagrams are complete and linked.
 - [ ] VPN Access: Ensure VPN Access is provided if required to run the workflow
 - [ ] Toggl: Ensure the Toggl project name is provided.
+
+## Pre-deployment Checklist(For consultants)
+
+Before closing off this issue, ensure the following items are checked:
+
+- [ ] **Run test suite**: For major releases that affect most parts of the workflow, run the entire test suite and ensure that it passes
+- [ ] **Version-locking of CLI**: Ensure that each job is version locked to the adaptor versions that are specified in the [cache registry](https://github.com/MSF-OCG/LIME-EMR/blob/main/scripts/run_msf_addons.sh#L53). 
```

Comment:
@AishaHassen I would call this `Adaptors version-locking` (rather than CLI version locking that's something else)